web/srv/api_handlers.go

```diff
@@ -242,3 +246,39 @@ func (h *handler) handleApiTap(w http.ResponseWriter, req *http.Request, p httpr
 		}
 	}
 }
+
+func (h *handler) handleProfileDownload(w http.ResponseWriter, req *http.Request, params httprouter.Params) {
+	service := req.FormValue("service")
+	namespace := req.FormValue("namespace")
+
+	if service == "" || namespace == "" {
+		err := fmt.Errorf("Service and namespace must be provided to create a new profile")
+		log.Error(err.Error())
```

Comment:
i realize we're not consistent about this in the codebase, but `log.Error(err)` will do the same thing here.